src/mpid/ch4/netmod/ofi/ofi_events.h

```diff
@@ -80,7 +80,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_recv_event(struct fi_cq_tagged_entry *wc,
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_OFI_RECV_EVENT);
 
     rreq->status.MPI_SOURCE = MPIDI_OFI_cqe_get_source(wc, true);
-    rreq->status.MPI_ERROR = MPI_SUCCESS;
+    MPIDI_OFI_idata_get_error_bits(wc->data, &rreq->status.MPI_ERROR);
```

Comment:
I'd like to see if we can make this API look more like `MPIDI_OFI_cqe_get_source` above it. I.e. make error the return value. Maybe the prefix could also be `MPIDI_OFI_cqe_get_`?

Reply:
👍 Done